keel-api/src/test/resources/examples/titus-cluster-example.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I'm not sure exactly what info is needed here (and what should be defaulted), but I tried to group things into some general buckets like `containerOptions`.

Reply:
the modeling itself looks pretty spiffy. Looking at what's listed here, it feels like we might be able to get to the point where most clusters accept the defaults for everything titus-specific besides `resources` and `container`.

Reply:
I bet we could default for resources too, and just allow you to specify if needed. I think we could get down to just specifying the container!